GPU/Vulkan/TextureScalerVulkan.cpp

```diff
@@ -21,38 +21,45 @@
 #endif
 
 #include <algorithm>
-#include "gfx/gl_common.h"
 
-#include "GPU/Common/TextureScalerCommon.h"
-#include "GPU/Vulkan/TextureScalerVulkan.h"
+#include "Common/Vulkan/VulkanContext.h"
 #include "Common/ColorConv.h"
 #include "Common/Log.h"
 #include "Common/ThreadPools.h"
+#include "GPU/Common/TextureScalerCommon.h"
+#include "GPU/Vulkan/TextureScalerVulkan.h"
+
+// TODO: Share in TextureCacheVulkan.h?
```

Comment:
I prefer GPU/Vulkan/VulkanUtils.cpp/h (which may be renamed or split later) for "ppsspp-specific" things. Ideally, Common/Vulkan should not know about PPSSPP concepts.